bridges/chains/chain-asset-hub-rococo/src/lib.rs

```diff
@@ -74,3 +81,40 @@ pub fn build_congestion_message<RuntimeCall>(
 
 /// Identifier of AssetHubRococo in the Rococo relay chain.
 pub const ASSET_HUB_ROCOCO_PARACHAIN_ID: u32 = 1000;
+
+/// AssetHubRococo parachain.
+#[derive(RuntimeDebug)]
+
+pub struct AssetHubRococo;
+
+impl Chain for AssetHubRococo {
```

Comment:
I would move these files somewhere in the folder where the chains are defined. Similar to how it's done here https://github.com/polkadot-fellows/runtimes/pull/627 for bridge-hub-kusama and bridge-hub-polkadot. @bkontur WDYT ?

Reply:
> I would move these files or at least part of their content somewhere in the folder where the chains are defined. Similar to how it's done here [polkadot-fellows/runtimes#627](https://github.com/polkadot-fellows/runtimes/pull/627) for bridge-hub-kusama and bridge-hub-polkadot. @bkontur WDYT ?

Hmm, actually, good idea, then the `Bridges` subfolder will contains just primitives/pallets and no chain related data. I like it. We have rococo/constants and westend/constants.

Could you do it with @antkve as a part of https://github.com/paritytech/parity-bridges-common/issues/2950 please?

But first, let's clean kusama/polkadot stuff out of here: https://github.com/paritytech/polkadot-sdk/pull/8057

Reply:
> Could you do it with @antkve as a part of https://github.com/paritytech/parity-bridges-common/issues/2950 please?

Sure, we can do it as a follow-up. I'll add it to the issue